.gitignore

```diff
@@ -0,0 +1,4 @@
+/venv
+db.sqlite3
+.idea
+.pyc
```

Comment:
It is recommended to ignore all `.pyc` files by using `*.pyc` instead of just `.pyc`. This ensures that all compiled Python files are ignored, regardless of their location in the project.